subsys/bluetooth/mesh/access.c

```diff
@@ -506,15 +506,37 @@ static bool model_has_dst(struct bt_mesh_model *mod, uint16_t dst)
 	return mod->elem_idx == 0;
 }
 
-static const struct bt_mesh_model_op *find_op(struct bt_mesh_model *models,
-					      uint8_t model_count, uint32_t opcode,
-					      struct bt_mesh_model **model)
+static const struct bt_mesh_model_op *find_op(struct bt_mesh_elem *elem,
+					      uint32_t opcode, struct bt_mesh_model **model)
 {
 	uint8_t i;
+	uint8_t count;
+	/* This value shall not be used in shipping end products. */
+	uint16_t cid = 0xffff;
```

Comment:
This can still happen though, as we can't control what other devices are doing. Instead, could this be `uint32_t cid = UINT32_MAX` or something, so we can guarantee correct behavior for all potential values?